homeassistant/components/nest/camera.py

```diff
@@ -1,9 +1,9 @@
 """Support for Nest cameras that dispatches between API versions."""
 
+from homeassistant.components.nest.legacy.camera import async_setup_legacy_entry
```

Comment:
We prefer relative imports for intra integration package imports.

Reply:
Fixed.  I left one where the legacy subirectory has an import `from homeassistant.components.nest import config_flow`.  If needed i can swap it for some other style import, or i can move the calls around if that isn't good.